src/AutoRest.CSharp/LowLevel/AutoRest/DpgOutputLibrary.cs

```diff
@@ -8,26 +8,26 @@
 
 namespace AutoRest.CSharp.Output.Models.Types
 {
-    internal class LowLevelOutputLibrary : OutputLibrary
```

Comment:
Will you change other LowLevel* class names? This is confusing. 

Is it better to keep the LowLevelOutputLibrary and add something like ConvenienceOutputLibrary which extends LowLevelOutputLibrary?

Reply:
> Will you change other LowLevel* class names? 

Yes. That would be a separate change though, this one is too big. 
DataPlaneOutputLibrary and related types will be renamed as well.